### PR TITLE
Minor bugfix/cleanup

### PR DESCRIPTION
**Pull request by pariator**
Maze Slimes in Twilight Forest were spawning with DoNotSpawnSlime tags (normal slimes seem fine still). This fixes that issue and handles the slimes slightly better.

### CODE REVIEW

**Review by draknyte1 — CHANGES_REQUESTED** [review]
You can’t cast to EntitySlime if it’s not an EntitySlime. You’ll crash out with Class Cast Exception.

**Comment by pariator** [comment]
> 

Thanks. I didn't catch that. I'll add a check in that part of the method.

File: src/main/java/gregtech/api/util/GT_SpawnEventHandler.java [hunk 1]
```diff
@@ -36,6 +41,7 @@ public void denyMobSpawn(CheckSpawn event) {
                         double dy = rep[1] + 0.5F - event.entity.posY;
                         double dz = rep[2] + 0.5F - event.entity.posZ;
                         if ((dx * dx + dz * dz + dy * dy) <= Math.pow(r, 2)) {
+                            ((EntitySlime) event.entityLiving).setCustomNameTag("DoNotSpawnSlimes");
```
Comment (draknyte1) [inline src/main/java/gregtech/api/util/GT_SpawnEventHandler.java:44]: Here.

**Comment by Dream-Master** [comment]
@pariator there is now some conflicts. Can you resolve it ? thanks

**Pull request merged**